blogposts/2022/release-3.37.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Sourcegraph 3.37 release'
+publishDate: 2022-02-22T10:00-07:00
+description: 'Sourcegraph 3.37 introduces performance improvements for Code Intelligence, sharing for Notebooks, and a new UI for creating search contexts.'
+tags: [blog, release]
+slug: 'release/3.37'
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.37/sourcegraph-3-37-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.37/sourcegraph-3-37-release.png
+changelogItems:
+  - description: 'Code in search results is now selectable for easier copying and reuse. You can still click on the code to open the corresponding file.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/30033
+    category: Search
+  - description: 'Query-based search contexts are now enabled by default as a beta feature, making it easier to scope queries for monorepos or large codebases.'
+    url: https://docs.sourcegraph.com/code_search/how-to/search_contexts#beta-query-based-search-contexts
+    category: Search
+  - description: 'GraphQL API: The order of events returned by `MonitorTriggerEventConnection` has been reversed so newer events are returned first. The `after` parameter has been modified accordingly to return events older the one specified, to allow for pagination.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/31219
+    category: API
+  - description: Fixed a race condition in the precise Code Intelligence upload expiry process that prematurely expired new uploads.
+    url: https://github.com/sourcegraph/sourcegraph/pull/30546
+    category: Code Intelligence
+  - description: 'Fixed performance issue in LSIF upload processing, reducing the latency between uploading an LSIF index and accessing precise Code Intelligence in the UI.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/31143
+    category: Code Intelligence
+---
+
+Sourcegraph 3.37 is now available! For this release, we introduced:
+
+## Code Intelligence LSIF indexing time has been reduced over 90%
```

Comment:
@iskyOS - "Code Intelligence LSIF indexing time has been reduced by over 90%" 

If we went from ~15 minutes to 30 seconds, could we also say 30x faster? Does that sound too big to be realistic?

Reply:
@abschu I don't think it's too big, I just wasn't really sure how the math worked to say "X times faster" for a reduction in time. Logically though you could index a file 30 times over in the time it took to index 1 previously, so I think that's accurate to say

Reply:
We could say this instead:

```suggestion
## Code Intelligence LSIF indexing is now 30x faster
```

Reply:
The math does work out. Maybe let's just say both?

```suggestion
## Code Intelligence LSIF indexing time has improved 30x (from 15m to 30s)
```

Reply:
Alternatively, we could include the specifics in the paragraph below.